process/interceptors/factory/nilFinalityAttester.go

```diff
@@ -0,0 +1,31 @@
+package factory
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/logger"
+)
+
+var log = logger.GetOrCreate("process/interceptors/factory")
+
+// nilFinalityAttester is a nil implementation that will cause all received headers
```

Comment:
that returns default value, without checking actual finality.

Reply:
implementation removed, no longer necessary